src/routes/+page.svelte

```diff
@@ -71,9 +71,11 @@
 
 <article class="layout-prose">
 	<p>
-		The {COMPETITION_FULL_NAME}
-		(SoME) is an annual competition to foster the creation of excellent math content online. On this
-		page, you can sign up as either a creator or judge for the {listFormatter.format(categories)} categories.
+		The <a href=https://www.3blue1brown.com/blog/some1> {COMPETITION_FULL_NAME} </a>
```

Comment:
Will this link work?

Reply:
Grant said he'll write a substack post instead to avoid pointing to an article with misaligned dates

Reply:
I see. No big deal then. I was just crossing off the final box to be ready for registration

Reply:
Also the formatting is wrong the quotes are missing. Could you just remove this link for now?